src/flash/display/MovieClip.ts

```diff
@@ -114,6 +114,32 @@ module Shumway.AVMX.AS.flash.display {
     }
   }
 
+  export interface FrameScript {
+    (any?): any;
+    precedence?: number;
+    context?: MovieClip;
```

Comment:
I like context more. Any other recommendations? We already use the term symbol for objects to initialize instances from, not the instances itself.